packages/jest-cli/src/cli/index.js

```diff
@@ -30,6 +30,7 @@ import {print as preRunMessagePrint} from '../pre_run_message';
 import runJest from '../run_jest';
 import Runtime from 'jest-runtime';
 import TestWatcher from '../test_watcher';
+import updateGlobalConfig from '../lib/update_global_config';
```

Comment:
Lint should have yelled at us for leaving this here. It printed it as a warning, not an error